app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+   <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
`Ruby on Rails Tutorial Sample App` 
tìm hiểu I18n và sử dụng i18n cho toàn bộ text của các file html